Gems/Atom/Feature/Common/Assets/Shaders/ImGui/ImGui.azsl

```diff
@@ -14,18 +14,20 @@ struct VertexInput
     float2 Position : POSITION;
     float2 UV : UV;
     float4 Color : COLOR;
+    uint TextureIndex : INSTANCE_DATA;
 };
 
 struct VertexOutput
 {
     float4 Position : SV_Position;
     float4 Color : COLOR;
     float2 UV : UV;
+    uint TextureIndex : INSTANCE_DATA;
 };
 
 ShaderResourceGroup ObjectSrg : SRG_PerObject
```

Comment:
This Srg should be a pass Srg and it should be only applied once.

Reply:
Done!